core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -808,10 +813,35 @@ public SystemSessionProperties(
                         featuresConfig.isHideInaccessibleColumns(),
                         value -> validateHideInaccessibleColumns(value, featuresConfig.isHideInaccessibleColumns()),
                         false),
+                integerProperty(
```

Comment:
those are usefult for experimenting - but maybe we should declare them as hidden?